Sources/KituraContracts/ClousureAliases.swift

```diff
@@ -220,10 +244,10 @@ router.get("/status") { (respondWith: (Status?, RequestError?) -> Void) in
 public typealias SimpleCodableClosure<O: Codable> = (@escaping CodableResultClosure<O>) -> Void
 
 /**
-The `IdentifierSimpleCodableClosure` is for use in cases where you'd want to perform a series of actions utilising an object which conforms to `Identifier` then respond with an object conforming to `Codable` or a `RequestError` in the form of a `CodableResultClosure`
+The `IdentifierSimpleCodableClosure` is for use in cases where you'd want to perform a series of actions utilising an object which conforms to `Identifier` then respond with an object conforming to `Codable` or a `RequestError` in the form of a `CodableResultClosure`.
```

Comment:
IdentifierCodableArrayClosure - the usage example text isn't displaying correctly - see below - apologies that I can't reference the correct line - 
"If there has been an error you can use the respondWith call to respond with an appropiate error and passing nil for the [User]?. ### Usage Example: ###"